### DISCUSSION TITLE
SRT stimuli

### DISCUSSION THREAD

**Discussion by vekteo** [jspsych/jsPsych#741] [Q&A] [answered]
Hi everybody,

I'm setting up an experiment with the SRT plugin. I'm wondering if there is a way to change the stimuli. For example, to present circles or images instead of the squares.

Thank you very much,

Teodora

**Comment by jodeleeuw** [comment, marked as answer]
Hi @vekteo,

Unfortunately that plugin doesn't have any parameters for controlling the stimuli. You can modify the plugin code directly to change how the stimulus is generated. There is a function at the bottom of the `jspsych-serial-reaction-time.js` plugin file called `plugin.stimulus()` that is responsible for creating the stimulus. That would be the place to start.

**Reply by vekteo** [reply]
Thank you very much!
